cirq/protocols/__init__.py

```diff
@@ -84,10 +84,9 @@
     SupportsQasmWithArgs,
     SupportsQasmWithArgsAndQubits,
 )
-from cirq.protocols.trace_distance_bound import (
-    SupportsTraceDistanceBound,
-    trace_distance_bound,
-)
+from cirq.protocols.trace_distance_bound import (SupportsTraceDistanceBound,
+                                                 trace_distance_bound,
+                                                 trace_distance_from_angle_list)
```

Comment:
Add a trailing comma to make yapf not do this.

Reply:
done